src/components/ChannelList/hooks/listeners/useChannelUpdated.js

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect } from 'react';
+
+import { ChatContext } from '../../../../context';
+
+export const useChannelUpdated = ({ onChannelUpdated, setChannels }) => {
+  const { client } = useContext(ChatContext);
+
+  useEffect(() => {
+    const handleEvent = (e) => {
+      if (onChannelUpdated && typeof onChannelUpdated === 'function') {
```

Comment:
made this comment in few places
```suggestion
      if (typeof onChannelUpdated === 'function') {
```